src/controllers/goals/goals.models.js

```diff
@@ -0,0 +1,28 @@
+const knex = require('../../data/dbConfig.js');
+
+const insert = async goal =>
+  knex('goals')
+    .insert(goal)
+
+const find = async id =>
+  knex('goals')
+    .where('id', id)
+    .first()
+
+const update = async (id, goal) =>
+  knex('goals')
+    .where('id', id)
+    .update(goal)
+
+const del = async id => {
```

Comment:
Same thing w/ the names, maybe just make the function names more specific. Just preference, don't feel like you have to.